packages/manager/modules/pci/src/projects/project/failover-ips/redirection.js

```diff
@@ -0,0 +1,11 @@
+export default /* @ngInject */ ($urlRouterProvider) => {
+  // redirect from "/failover-ips" to "/additional-ips"
+  // #/pci/projects/{project-id}/failover-ips
+
+  $urlRouterProvider.when(/^\/pci\/projects\/.*\/failover-ips/, () => {
```

Comment:
**question**
Shouldn't this be defined in the parent module? As this module is lazyloaded, this route catching may not be active on the right moment no?